oddbird/oddvest
# Switch to fetch() and add a test

## Cute animal pic
_Because everyone likes pictures of animals._

![](https://www.audubon.org/sites/default/files/styles/grid_gallery_lightbox/public/Northern_Saw-whet_Owl_b13-38-546_l.jpg)

## Description
_The commit messages say what you did; this should explain why and how._

Just sending smaller PRs for a faster review cycle. This one just converts getHarvestJSON to use `fetch()` and adds a test for it.

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally._

Set up ngrok, enabled "Oddvest - Carl Dev" on OddMinistrative board, added/removed a task from a card, everything still works.

No functional change expected from this PR.

---

## Reviewer tasks:
